### PR TITLE
BaSP-TG: Super Admin section

### PR DESCRIPTION
**Pull request by SteampunkMatu**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-import getAllAdmins from './resources/admins';
+import {
+  postSuperAdmins, deleteSuperAdmins,
+} from './resources/super-admins';
 
 const app = express();
 const port = process.env.PORT || 3000;
 
 app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
```
Comment (lmalvasia) [inline src/index.js:11]: Why you added this change?

File: src/resources/super-admins.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const user = {
+    id: req.body.id,
+    email: req.body.email,
+    first_name: req.body.first_name,
+    last_name: req.body.last_name,
+    date_of_birth: req.body.date_of_birth,
+    DNI: req.body.DNI,
+    Phone: req.body.Phone,
+  };
+  superAdmins.push(user);
```
Comment (lmalvasia) [inline src/resources/super-admins.js:14]:
```suggestion
  const newSuperAdmins = superAdmins.push(user);
```

File: src/resources/super-admins.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  fs.writeFile('src/data/super-admins.json', JSON.stringify(superAdmins), () => {
```
Comment (lmalvasia) [inline src/resources/super-admins.js:15]:
```suggestion
  fs.writeFile('src/data/super-admins.json', JSON.stringify(newSuperAdmins), () => {
```

File: src/resources/super-admins.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    if (!user.email || !user.first_name || !user.last_name
```
Comment (lmalvasia) [inline src/resources/super-admins.js:16]: You have to execute this validation before writing inside the file

File: src/resources/super-admins.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+          || !user.date_of_birth || !user.DNI || !user.Phone) {
+      res.send('Cannot save new user');
```
Comment (lmalvasia) [inline src/resources/super-admins.js:18]:
```suggestion
      res.status(404).json({
        error: 'Cannot create a super admin. Invalid body params'
      });
```

File: package.json [hunk 6]
```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "express": "^4.18.0",
+    "lint": "^0.7.0",
```
Comment (lmalvasia) [inline package.json:29]: This is strange. Why did you change this?

**Review by lmalvasia — COMMENTED** [review]
Changes and questions

File: package.json (hunk 6 above)
Comment (lmalvasia) [reply package.json:29]: Remove this new dependency and run `npm install` again.

File: src/index.js (hunk 1 above)
Comment (SteampunkMatu) [reply src/index.js:11]: That's was a line to test some expressions, i forgot to delete it

File: package.json (hunk 6 above)
Comment (SteampunkMatu) [reply package.json:29]: I dont know where that line come from, should i delete it?

**Review by lmalvasia — APPROVED** [review]
